src/iterators/ParticleIteratorWrapper.h

```diff
@@ -0,0 +1,65 @@
+/**
+ * ParticleIteratorWrapper.h
+ *
+ *  Created on: 29 May 2018
+ *      Author: seckler
+ */
+#pragma once
+
+#include "ParticleIteratorInterface.h"
+
+namespace autopas {
+
+/**
+ * ParticleIteratorInterface class to iterate over particles.
```

Comment:
Wrong doc. Here also the necessity for a wrapper should be explained in a way someone new to the code understands it.